docs/distributed/README.md

```diff
@@ -43,7 +43,7 @@ To start a distributed Minio instance, you just need to pass drive locations as
 *Note* 
 
 - All the nodes running distributed Minio need to have same access key and secret key for the nodes to connect. To achieve this, you need to export access key and secret key as environment variables on all the nodes before executing Minio server command.
-- Disks used for Minio distributed should be fresh with no pre-existing data. 
+- Minio distributed mode requires fresh directories. If required, the drives can be shared with other applications. You can do this by using a sub-directory exclusive to minio. For example, if you have mounted your volume under `/export`, pass `/export/data` to Minio server arguments.
```

Comment:
, pass /export/data as arguments to Minio server.

Reply:
updated @deekoder